cuizinart/frontend/src/components/map.jsx

```diff
@@ -16,7 +16,7 @@ L.Icon.Default.mergeOptions({
 });
 
 //L.Marker.prototype.options.icon = DefaultIcon;
-const stamenTonerTiles = 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png';
+const stamenTonerTiles = 'https://maps.wikimedia.org/osm-intl/{z}/{x}/{y}.png';
 const stamenTonerAttr = '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>';
```

Comment:
Please add an attribution to Wikimedia and OSM contributors, like so: https://foundation.wikimedia.org/wiki/Maps_Terms_of_Use#/media/File:Screenshot_of_a_map_on_Wikivoyage_displaying_San_Francisco's_Financial_District.png